Jenkinsfile

```diff
@@ -1,7 +1,18 @@
-// Readme @ http://gitlab.iex.ec:30000/iexec/jenkins-library
-@Library('global-jenkins-library@1.6.2') _
+@Library('global-jenkins-library@feature/standalone-scone-build') _
 
-def nativeImage = buildSimpleDocker_v2(dockerfileDir: './docker', buildContext: '.',
-        dockerImageRepositoryName: 'tee-worker-pre-compute', imageprivacy: 'public')
-sconeBuildAllTee(nativeImage: nativeImage, targetImageRepositoryName: 'tee-worker-pre-compute',
-        sconifyArgsPath: './docker/sconify.args')
+buildInfo = getBuildInfo()
+
+def nativeImage = buildSimpleDocker_v2(
+  buildInfo:                 buildInfo,
+  dockerfileDir:             './docker',
+  buildContext:              '.',
+  dockerImageRepositoryName: 'tee-worker-pre-compute',
+  imageprivacy:              'private'
+)
+
+sconeBuildUnlocked(
+  nativeImage:     nativeImage,
+  imageName:       'tee-worker-pre-compute',
```

Comment:
```suggestion
  imageName:       imageName,
```

Reply:
Do you mean to add an `imageName` variable and to use it in both `sconeBuildUnlocked` and `buildSimpleDocker_v2` ?

Reply:
Yes